assets/monitoring/teleport.service

```diff
@@ -7,7 +7,7 @@ Type=simple
 Restart=always
 User=ubuntu
 LimitNOFILE=32000
-ExecStart=/home/ubuntu/go/src/github.com/gravitational/teleport/build/teleport start --config=/etc/teleport/teleport.yaml  --gops --gops-addr=127.0.0.1:4321 --diag-addr=127.0.0.1:3434
+ExecStart=/home/ubuntu/go/src/github.com/gravitational/teleport/build/teleport start --config=/etc/teleport/teleport.yaml  -d --diag-addr=127.0.0.1:3434
```

Comment:
change port number here too

Reply:
I was going to add a note, apart from legacy reasons.. any reason why we still running the marketplace on 3434? https://github.com/gravitational/teleport/search?q=3434&unscoped_q=3434 @webvictim

Reply:
I won't change the marketplace image in this PR but we should come back and review it. Maybe look at shipping these to CloudWatch.